examples/quad_android/src/lib.rs

```diff
@@ -0,0 +1,935 @@
+#[cfg(not(any(
```

Comment:
does this have to be a separate example? could we make it a part of the existing `quad` instead?